### PR TITLE
Remove temp_storage from update

### PR DESCRIPTION
**Pull request by dafeda**
## Pre review checklist

- [ ] Read through the code changes carefully after finishing work
- [ ] Make sure tests pass locally (after every commit!)
- [ ] Prepare changes in small commits for more convenient review (optional)
- [ ] PR title captures the intent of the changes, and is fitting for release notes.
- [ ] Updated documentation
- [ ] Ensured that unit tests are added for all new behavior (See 
    [Ground Rules](https://github.com/equinor/ert/blob/main/CONTRIBUTING.md#ground-rules)),
    and changes to existing code have good test coverage.

## Pre merge checklist
- [x] Added appropriate release note label
- [x] Commit history is consistent and clean, in line with the [contribution guidelines](https://github.com/equinor/ert/blob/main/CONTRIBUTING.md).

### CODE REVIEW

File: src/ert/analysis/_es_update.py [hunk 1]
```diff
@@ -175,66 +178,49 @@ def _param_ensemble_for_projection(
...
+                    "names": [e.name for e in config_node.transfer_functions],
+                }
+            )
+            target_fs.save_parameters(param_group, realization, dataset)
+        elif isinstance(config_node, (Field, SurfaceConfig)):
+            target_fs.save_parameters(
+                param_group, realization, da.sel(realizations=realization).to_dataset()
+            )
+        else:
+            raise NotImplementedError(f"{type(config_node)} is not supported")
```
Comment (oyvindeide) [inline src/ert/analysis/_es_update.py:223]: Think we can safely get rid of this, as it is quite unlikely that another node will get in here 😅  Probably a remnant from when all nodes were in the same list, responses and parameters.

File: src/ert/analysis/_es_update.py [hunk 2]
```diff
@@ -244,31 +230,8 @@ def _create_temporary_parameter_storage(
     param_group: str,
 ) -> TempStorage:
     temp_storage = TempStorage()
-    t_genkw = 0.0
-    t_surface = 0.0
-    t_field = 0.0
-    _logger.debug("_create_temporary_parameter_storage() - start")
```
Comment (oyvindeide) [inline src/ert/analysis/_es_update.py:250]: This proved a bit useful earlier today, so perhaps add some kind of info logging? Could be on `save_parameters` or inside `_save_temp_storage_to_disk`

**Review by oyvindeide — APPROVED** [review]
LGTM! Just some minor comments

**Comment by dafeda** [comment]
I think we need to think about this a bit as it really is not nice to have to index arrays differently and do transposes.
Perhaps we should re-consider how we store the data in storage.
Closing for now.

**Pull request closed without merging**